lib/client/components/Location.jsx

```diff
@@ -11,17 +12,18 @@ export default class Location extends React.Component {
   }
 
   handleClick() {
-    navigator.geolocation.getCurrentPosition(
+    this.setState({ location: 'Fetching location...', error: '' });
+
+    Geolocation.getCurrentPosition(
       geo => {
         const lat = geo.coords.latitude;
         const lon = geo.coords.longitude;
 
-        this.setState({ lat, lon, location: `${lat}, ${lon}`, error: '' });
+        this.setState({ lat, lon, location: `${lat.toFixed(4)}, ${lon.toFixed(4)}`, error: '' });
```

Comment:
I did this for a couple of reasons, both related to display.  (Note that the latitude and longitude are still set to whatever the device provides.)  First, it makes the output symmetric.  Nothing irritates me more than a string like "3.5, 4.25" when it's easy to make "3.50, 4.25".  Personal preference.  :)  Also, this prevents the data from marching off the screen.  There were a ridiculous number of digits coming back from my browsers and devices.